modules/gdscript/tests/scripts/runtime/features/setter_getter_with_name.gd

```diff
@@ -0,0 +1,99 @@
+	print(get("var_with_typed_normal_setget"))
+	print(get("var_with_untyped_named_setget"))
+	print(get("var_with_typed_named_setget"))
+	print(get("static_var_with_untyped_normal_setget"))
+	print(get("static_var_with_typed_normal_setget"))
+	print(get("static_var_with_untyped_named_setget"))
+	print(get("static_var_with_typed_named_setget"))
+	var_with_untyped_normal_setget = 1
+	var_with_typed_normal_setget = 1
+	var_with_untyped_named_setget = 1
+	var_with_typed_named_setget = 1
+	static_var_with_untyped_normal_setget = 1
+	static_var_with_typed_normal_setget = 1
+	static_var_with_untyped_named_setget = 1
+	static_var_with_typed_named_setget = 1
+	set("var_with_untyped_normal_setget", 2)
+	set("var_with_typed_normal_setget", 2)
+	set("var_with_untyped_named_setget", 2)
+	set("var_with_typed_named_setget", 2)
+	set("static_var_with_untyped_normal_setget", 2)
+	set("static_var_with_typed_normal_setget", 2)
+	set("static_var_with_untyped_named_setget", 2)
+	set("static_var_with_typed_named_setget", 2)
+
+func set_normal_untyped(value, optional = true):
+	prints("set_normal_untyped", value, optional)
+
+func get_normal_untyped(optional = true):
+	prints("get_normal_untyped", optional)
+	return 0
+
+func set_normal_typed(value: int, optional: bool = true) -> void:
+	prints("set_normal_typed", value, optional)
+
+func get_normal_typed(optional = true) -> int:
+	prints("get_normal_typed", optional)
+	return 0
+
+func set_named_untyped(name, value, optional = true):
+	prints("set_named_untyped", name, value, optional)
+
+func get_named_untyped(name, optional = true):
+	prints("get_named_untyped", name, optional)
+	return 0
+
+func set_named_typed(name: StringName, value: int, optional: bool = true) -> void:
+	prints("set_named_typed", name, value, optional)
+
+func get_named_typed(name: StringName, optional: bool = true) -> int:
+	prints("get_named_typed", name, optional)
+	return 0
+
+static func static_set_normal_untyped(value, optional = true):
+	prints("static_set_normal_untyped", value, optional)
+
+static func static_get_normal_untyped(optional = true):
+	prints("static_get_normal_untyped", optional)
+	return 0
+
+static func static_set_normal_typed(value: int, optional: bool = true) -> void:
+	prints("static_set_normal_typed", value, optional)
+
+static func static_get_normal_typed(optional = true) -> int:
+	prints("static_get_normal_typed", optional)
+	return 0
+
+static func static_set_named_untyped(name, value, optional = true):
+	prints("static_set_named_untyped", name, value, optional)
+
+static func static_get_named_untyped(name, optional = true):
+	prints("static_get_named_untyped", name, optional)
+	return 0
+
+static func static_set_named_typed(name: StringName, value: int, optional: bool = true) -> void:
+	prints("static_set_named_typed", name, value, optional)
+
+static func static_get_named_typed(name: StringName, optional: bool = true) -> int:
+	prints("static_get_named_typed", name, optional)
+	return 0
```

Comment:
```suggestion
	return 0

```
Missing termination, not enforced but neater with it

Reply:
Also this test is hard to read in my opinion.
@nlupugla I prepared another test, let me know what you think about it: [gist](https://gist.github.com/dalexeev/a694f8306d1ecdf429ad8769eb105048).

Reply:
I like that layout much better! I was thinking that the test was hard to read, which is why I changed the variable names from "a, b, c, d" to something more descriptive. I'll commit the test later today.